.changeset/real-oranges-rescue.md

```diff
@@ -0,0 +1,20 @@
+---
+'skuba': minor
+---
+
+migrate: Introduce `skuba migrate node-version` to automatically upgrade a project's Node.js version
+
+`skuba migrate node-version` will attempt to automatically upgrade projects to Node.js 20.
```

Comment:
```suggestion
`skuba migrate node-version` will attempt to automatically upgrade projects to Node.js 20.
```

question: should this be specifically node20 instead?

Reply:
I was thinking of making it sufficiently general for future releases, we could potentially just make the number a required argument? Thoughts?